tfhe/src/integer/bigint/algorithms.rs

```diff
@@ -182,25 +223,89 @@ where
     (quotient, remainder)
 }
 
+pub(crate) fn slow_div_signed<T>(numerator: T, divisor: T) -> (T, T)
+where
+    T: Numeric
+        + ShlAssign<u32>
+        + Shl<u32, Output = T>
+        + Shr<u32, Output = T>
+        + BitOrAssign<T>
+        + SubAssign<T>
+        + BitAnd<T, Output = T>
+        + Ord
+        + Neg<Output = T>
+        + Add<T, Output = T>
+        + Sub<T, Output = T>,
```

Comment:
don't we have a signed trait ? SignedNumeric IIRC ?

Reply:
Yes but putting `T:SignedInteger` already implements division, however this is used to implement division on T